File: mario.c
// Creates 2 mario-like piramids
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Asks for height
    int h;
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);
    
    // Prints hashes
    for (int i = 0; i < h; i++)
    {
        for (int k = i; k < h - 1; k++)
        {
            printf(" ");
        }
        for (int j = -1; j < i; j++)
        {
            printf("#");
        }
        for (int l = 1; l < 2; l++)
        {
            printf("  ");
        }
        for (int z = h - i - 1; z < h; z++)
        {
            printf("#");
        }
        printf("\n");
    }       
}

File: mario.py
# import get int from cs50 library and get int
from cs50 import get_int

# loop for proper height
while True:
    h = get_int("Height: ")
    if h >= 1 and h <= 8:
        break

# iterate over a height
for i in range(h):
    # print spaces and then hashes
    print(" " * (h - 1 - i), end="")
    print("#" * (i + 1), end="")
    # print additional space and hashes in reverse order
    print(" ", "#" * (i + 1), end="")
    # print new line
    print()
